docker/generate_makefile.py

```diff
@@ -37,12 +37,15 @@ def _print_makefile_run_template(image):
     fuzzer = image['fuzzer']
     benchmark = image['benchmark']
 
-    for run_type in ('run', 'debug', 'test-run'):
+    for run_type in ('run', 'debug', 'test-run', 'repro-check'):
+        if run_type == 'repro-check':
+            bench_path = os.path.join(BENCHMARK_DIR, benchmark, 'testcases')
+            if not os.path.isdir(bench_path):
+                continue
```

Comment:
i think we need to exception here saying, "This is not a bug-based benchmark, testcases dir missing!"

Reply:
But isn't this run for all benchmarks? If it's not a bug based benchmark we simply don't create the make rule to run the reproducibility check, otherwise we will get exceptions every time the makefile is generated

Reply:
ah ok, sg then, no change needed.